src/pages/preferences/ProfileTab.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useState } from "react";
+import styled from "@emotion/styled";
+import Card from "@leafygreen-ui/card";
+import TextInput from "@leafygreen-ui/text-input";
+import Button, { Variant } from "@leafygreen-ui/button";
+import { Body } from "@leafygreen-ui/typography";
+import { Select } from "antd";
+import get from "lodash/get";
+import { timeZones, awsRegions } from "./fieldMaps";
+import { GithubUser } from "gql/generated/types";
+const { Option } = Select;
+
+interface ProfileTabProps {
+  githubUser?: GithubUser;
+  timezone?: string;
+  region?: string;
+}
+export const ProfileTab: React.FC<ProfileTabProps> = ({
+  githubUser,
+  timezone,
+  region,
+}) => {
+  const lastKnownAs = get(githubUser, "githubUser.lastKnownAs", "");
+  const [timezoneField, setTimezoneField] = useState(timezone || "UTC");
+  const [regionField, setRegionField] = useState(region || "us-east-1");
```

Comment:
these are good places to use nullish coalescing operator

Reply:
Cool didnt know about this.